test/tests/functional/pbs_power_provisioning_cray.py

```diff
@@ -97,9 +99,8 @@ def setup_cray_eoe(self):
                                     self.names[i])
 
         # Find nid range for capmc command
-        nids = str(min_nid) + "-" + str(max_nid)
         cmd = "/opt/cray/capmc/default/bin/capmc "
-        cmd += "get_power_cap_capabilities --nids=" + nids
+        cmd += "get_power_cap_capabilities --nids " + ','.join(nids)
```

Comment:
It will be a wee bit faster if you remove cmd += and put  \\ after the first string.  Python is super fast concatenating multiple string literals together.  Using + or += means the string needs to be regenerated.